Production_scripts/Create_slf_lookups/Produce_slf_lookup_tests/lookup_ltc_tests.R

```diff
@@ -0,0 +1,38 @@
+####################################################
+# Name of file lookup_ltc_tests.R
+# Original Authors - Zihao Li
+# Original Date - September 2022
+#
+# Written/run on - RStudio Server
+# Version of R - 3.6.1
+#
+# Description - Produce tests for source linkage files
+# lookup ltc files
+####################################################
+
+# Packages
+library(createslf)
+library(openxlsx)
+
+fyear <- "1920"
+
+
+# Read in Data---------------------------------------
+
+# Create new and old dataframes with measures for testing
+new_tests <- produce_slf_ltc_tests(readr::read_rds(get_slf_ltc_path(fyear)))
+old_tests <- produce_slf_ltc_tests(readr::read_rds(get_slf_ltc_path(update = previous_update()))) ## where is the old_update
```

Comment:
Not sure where the old update is. This is also linked to the function script get_slf_ltc_path().